e2e/native-esm-wasm/answer.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as wasm from './answer.wasm';
```

Comment:
This file is borrowed from https://github.com/hasharchives/wasm-ts-esm-in-node-jest-and-nextjs/tree/main/wasm-package. It’s a build artifact from https://github.com/hasharchives/wasm-ts-esm-in-node-jest-and-nextjs/tree/main/wasm-crate, which is not ideal. I’m not sure if bringing the whole `wasm-pack` process is worth it though.

Reply:
Replaced with `add(i32, i32)` from  [mdn/webassembly-examples](https://github.com/mdn/webassembly-examples/tree/c05c6f55f0522da8e698b67bc94f87b0442ec05c/understanding-text-format) in https://github.com/facebook/jest/pull/13505/commits/79e0f40d9397acb923595aa7c9070c24260f4274.